src/Toolkit/I18n.php

```diff
@@ -225,6 +247,13 @@ public static function translateCount(string $key, int $count, string $locale =
                 $message = $translation[2];
         }
 
+        if (!$locale) {
+            $locale = static::locale();
+        }
+
+        $formatter = static::decimalNumberFormatter($locale);
+        $count = $formatter->format($count);
```

Comment:
Because `$formatter` might be `null` according to my suggestion above, there should be a check for `null` here. The number should only be formatted if the `$formatter !== null`.